docs/odcs.rst

```diff
@@ -24,84 +24,188 @@ idea is to not require the user to provide a yum repo url when building
 container images.  Instead, OSBS will use ODCS to generate a yum repo with the
 relevant signed packages based on user configuration.
 
+Concepts
+--------
+
+Signing Intent
+""""""""""""""
+
+When an ODCS compose is used, the idea of a signing intent is used. This refers
+to the signature of the content to be included in the container image. For
+instance, an environment may provide the following intents: ``release``,
+``beta``, and ``unsigned``. Each one of those intents is then mapped to a list
+of signing keys. These signing keys are then used during compose creation. The
+packages to be included must have been signed by either one of the keys listed.
+In the example above, the intents could be mapped to the following keys::
+
+    # Only include packages that have been signed by "my-release-key"
+    release -> my-release-key
+    # Include packages that have been signed by either "my-beta-key" or
+    # "my-release-key"
+    beta -> my-beta-key, my-release-key
+    # Do not check signature of packages - may include unsigned packages
+    unsigned -> <empty>
+
+*This mapping will be defined in reactor-config-secret which will allow updates
```

Comment:
Maybe call this "the secret named by reactor_config_secret".

Reply:
Would it make sense to have an example for this secret with these changes?

Reply:
Replaced `reactor-config-secret` with suggested text.

Added example yaml.